tests/performace_tests/enkf/test_load_state.py

```diff
@@ -0,0 +1,54 @@
+import pytest
+
+# from libres_utils import ResTest
+from ecl.util.test import ExtendedTestCase
```

Comment:
Idealy we would not use anything from `ecl.util.test`, and preferably use native `pytest` instead of `unittest`, but the latter is more my opinion I suppose.

Reply:
Test will probably be removed from this PR. See https://github.com/equinor/ert/pull/2913#issuecomment-1042193068